test/Datadog.Trace.Tests/Configuration/EnvironmentConfigurationSourceTests.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using Datadog.Trace.Configuration;
+using Xunit;
+
+namespace Datadog.Trace.Tests.Configuration
+{
+    public class EnvironmentConfigurationSourceTests
+    {
+        [Theory]
+        [InlineData("DD_EnvironmentConfigurationSourceTests", "1", "1")]
```

Comment:
Can you also add a specific case for the integration settings since that is usually where our customers hit this behavior?